README.md

```diff
@@ -391,13 +395,43 @@ impl PyIterProtocol for MyCoroutine {
     }
 }
 
+// Exposing our custom awaitable to Python.
+// This will behave like a coroutine.
+#[pyfunction]
+fn my_coroutine() -> PyResult<MyCoroutine> {
+    Ok(MyCoroutine {})
+}
+
 // Lets just call our module await_from_rust for simplicity.
 #[pymodule]
 fn await_from_rust(_py: Python, m: &PyModule) -> PyResult<()> {
     m.add_class::<MyCoroutine>()?;
+    m.add_function(wrap_pyfunction!(my_coroutine, m)?).unwrap();
     Ok(())
 }
 ```
 
+And now we can call our Rust coroutine from Python.
+
+```py
+# python 3.8
+import asyncio
+
+import await_from_rust
+
+
+async def main():
+    # Everything works as if it was coroutine...
+    result = await await_from_rust.my_coroutine()
+    print(f"my_coroutine returned with: {result!r}"
+
+# But note that this won't work:
+# asyncio.run(await_from_rust.my_coroutine())
+# because asyncio won't recognise it as a `coroutine` object
+
+asyncio.run(main())
+```
+
+### Using Rust's futures as Python awaitables
```

Comment:
I'm just teasing a bit :wink:  This is what I'd like to do be able to do in the end. I can remove it if that's not where you want to take this examples page though.